util.go

```diff
@@ -73,3 +74,7 @@ func addrEqual(a, b net.Addr) bool {
 
 	return aType == bType && aIP.Equal(bIP) && aPort == bPort
 }
+
+func rndCandidateID() string {
+	return fmt.Sprintf("candidate:%d", time.Now().UnixNano())
```

Comment:
What if this is called concurrently? Maybe we should use a UUID?

Reply:
yes, that would work too, I'm using nano so I think a clash is very unlikely, but we could use a uuid library too

Reply:
@hugoArregui we could use https://github.com/pion/ice/blob/master/mdns.go#L31

If you want to move it into `util.go` and have the caller append `.local`

Reply:
good call! done